src/plots/plots.js

```diff
@@ -1510,7 +1512,7 @@ plots.transition = function(gd, data, layout, traces, frameOpts, transitionOpts)
             delete layoutUpdate[attr].range;
         }
 
-        Lib.extendDeepNoArrays(gd.layout, layoutUpdate);
+        plots.extendLayout(gd.layout, layoutUpdate);
```

Comment:
Unless there are bugs, yes. It iterates through each of those container arrays as a way of bypassing the `No` in `extendDeepNoArrays`. The option to force redraw makes non-animatable things Just Work.